src/colorprint.cc

```diff
@@ -22,14 +22,16 @@
 #include <string>
 
 #include "check.h"
-#include "internal_macros.h"
+#include "internal_macros.h"  // IWYU pragma: keep
 
 #ifdef BENCHMARK_OS_WINDOWS
 #include <windows.h>
 #include <io.h>
-#else
+#endif  // ifdef BENCHMARK_OS_WINDOWS
+
+#ifndef BENCHMARK_OS_WINDOWS
```

Comment:
why is this better than #else ?

Reply:
It's indifferent in this place, but elsewhere it was closer to 
```
#if one
...
#elseif
#if two
...
#endif
#endif
```
and i think you can agree that is getting problematic to read, as compared with
```
#if one
..
#endif

#if !one && two
...
#endif
```

Reply:
the latter is fine, but here i think the else is clearer. maybe we can eshrine this as "avoid #else .. #if ?